app/models/manageiq/providers/openstack/cloud_manager/host_aggregate/operations.rb

```diff
@@ -0,0 +1,37 @@
+module ManageIQ::Providers::Openstack::CloudManager::HostAggregate::Operations
```

Comment:
Ahh, ok. I'm happy to remove this. I was just following the convention in the other models, but if it's unnecessary I'll move it to the main class.